src/block/table-of-contents/schema.js

```diff
@@ -128,6 +128,32 @@ export const attributes = ( version = VERSION ) => {
 		versionDeprecated: '',
 	} )
 
+	attrObject.add( {
+		attributes: {
+			titleText: {
+				source: 'html',
+				selector: '.stk-table-of-contents__title',
+				default: 'Table of Contents',
```

Comment:
Please make this string translatable
```suggestion
				default: __( 'Table of Contents', i18n ),
```